website/blog/new-graphback-release-0-14.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Improving the GraphQL experience with Graphback 0.14
+author: Manyanda Chitimbo
+author_title: Software Engineer
+author_url: https://github.com/machi1990
+author_image_url: https://avatars3.githubusercontent.com/u/10106536?s=460&u=ba1f66c0295c5151f147dd869a543bdc68671c56&v=4
+tags: [graphql, graphback, graphqlcrud, graphql-js, nodejs, release]
+image: https://graphback.dev/img/logo.png
+---
+
+## TL;DR:
+Over the past couple of months, we have been working on improving Graphback. We are excited to announce that we have released Graphback 0.14.0, bringing a new and improved CRUD and runtime experience.
+Most notable changes in this release:
+
+- Full implementation of the CRUD API specification
+- Improved runtime API to remove complexity of setting up a Graphback API
+- RBAC Authorization on top of the Graphback Service and Keycloak
+
+The complete changelog is available at https://graphback.dev/docs/releases#0140
+
+<!--truncate-->
+
+## Introduction to Graphback
+
+<div style={{textAlign: 'center', width: '400px', height: '400px'}}>
```

Comment:
![image](https://user-images.githubusercontent.com/11743717/88152714-55ce4500-cbfc-11ea-8bbf-60ac4e695128.png)

It does not render in the centre.

Reply:
Yeap, but the div is center aligned. Not sure why 
![Screenshot from 2020-07-22 10-35-58](https://user-images.githubusercontent.com/10106536/88154587-57e9d100-cc07-11ea-8f73-8e0d02c21cc5.png)

/cc @kingsleyzissou any ideas?

Reply:
You could use an `img` html tag instead?

Reply:
@machi1990 let me have a look